woocommerce/woocommerce
# Update the automatically generated changelog filename to include PR number

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR updates the automatically generated changelog file to include the PR number. This helps guarantee uniqueness. I've opted to prepend the PR number to the branch name (which is what we were previously using), so that if, for example, your branch name were `fix/12345`, and your PR number is `23456`, the resulting change file should be `23456-fix-12345`.





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fork of this repo, if you don't have one already. Merge the changes from this PR to `trunk` of your fork.
2. Make sure that actions are enabled in your fork. 
3. Create a new PR in your fork, targeting your fork's `trunk` branch. The PR should change something in a project that has a changefile (such as Woo core). Select the option to automatically generate a change file in the PR template.
4. Verify that the change generation action runs, and verify that the created PR includes the PR number.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
